spec/controllers/questions_controller_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'rails_helper'
+
+describe QuestionsController do
+  let(:question) { create(:question) }
+
+  describe 'GET #index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.